src/joinRates/JoinRates.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useMemo, useState, useCallback } from 'react'
+
+import JoinRatesTable from './JoinRatesTable'
+import { JoinRatesProps } from './model'
+import { Quality } from '../quality/Quality'
+
+function getTableNames(tableNames?: Array<string>): { 'Table names': Array<string> } | {} {
+  return tableNames ? { 'Table names': tableNames } : {}
+}
+
+const JoinRates = ({
+  data,
+  joinRateId,
+  onRowClickHandler,
+  selectedRules,
+  selectRules,
+  mode,
+  setMode,
+  pagination,
+  smallSize = true,
+}: JoinRatesProps): JSX.Element => {
+  const [_id, _setId] = useState<string>(joinRateId || (Array.isArray(data) ? data[0].id : data.id))
+  const joinRatesData = useMemo(
+    () =>
+      Array.isArray(data)
+        ? data.map(dataItem => ({
+            key: dataItem.id,
+            id: dataItem.id,
+            ...getTableNames(dataItem.tableNames),
+            ...dataItem.dq_join_statistics,
+          }))
+        : [
+            {
+              key: data.id,
+              id: data.id,
+              ...getTableNames(data.tableNames),
+              ...data.dq_join_statistics,
+            },
+          ],
+    [data]
+  )
+  const JoinRatesTableColumns = useMemo(() => {
+    const columns = Array.isArray(data)
+      ? Object.keys(
+          data.reduce(
+            (acc: any, dataItem) => ({
+              ...getTableNames(dataItem.tableNames),
+              ...acc,
+              ...dataItem.dq_join_statistics,
```

Comment:
Just to mention.
Now we receive `dqJoinStatisticsJson` and `dqJoinResultJson` on the DQD.
Seems that we need to change this to `dq_join_statistics` and `dq_result` on the DQD (FE or BE).
Let's discuss.